data/campaigns/The_Rise_Of_Wesnoth/scenarios/01_A_Summer_of_Storms.cfg

```diff
@@ -157,7 +157,7 @@
         [/message]
         [message]
             speaker=King Eldaric IV
-            message= _ "You’re showing initiative, son! I’m proud of you! Yes, you may lead our forces to battle, it is time — but I’ll stay near to keep an eye on you. There is more to this raid than meets the eye, I think."
+            message= _ "You’re showing initiative, son! I’m proud of you! Yes, you may lead our forces in battle, it is time — but I’ll stay near to keep an eye on you. There is more to this raid than meets the eye, I think."
```

Comment:
I question this replacement; it seems to change the meaning and possibly not in a good way.

Reply:
Rationale is it matches the line that comes immediately before it.
https://github.com/wesnoth/wesnoth/blob/dd3baa488150bfc8d8cd0105da9083009cb29ca0/data/campaigns/The_Rise_Of_Wesnoth/scenarios/01_A_Summer_of_Storms.cfg#L154-L161
            message= _ "Father, I am of age now. May I lead our forces in battle?"

Reply:
Yeah, I guess that does make sense. I might've changed Haldric's preposition instead, though. But I'm not entirely sure which is better…

Reply:
I propose making both lines "lead our forces into battle". This good?